index.html

```diff
@@ -353,14 +353,22 @@ <h1>Important Terms</h1>
 			<dd>
 			  <p>Usable by users in ways they can control. References in this document relate to <cite><a href="https://www.w3.org/TR/WCAG21/#operable"><abbr title="Web Content Accessibility Guidelines">WCAG</abbr> 2.1 Principle 2: Content must be operable</a></cite> [[WCAG21]]. See <a>Keyboard Accessible</a>.</p>
 			</dd>
-			<dt><dfn data-export="" data-dfn-for="ARIA" data-local-lt="owned|owned element's">Owned Element</dfn></dt>
+			<dt><dfn data-export="" data-lt="owned child|child element|child|children|child elements">Owned Child</dfn></dt>
+			<dd>
+				<p>An 'owned child' is a direct <abbr title="Document Object Model">DOM</abbr> child of the <a>element</a>, a descendant of the <a>element</a> with only <rref>generic</rref> or <rref>presentation</rref> elements intervening, or a child specified via an <pref>aria-owns</pref> relationship to the element.</p>
```

Comment:
The generic element may still be included in the accessibility tree, but it should not disrupt the association between the parent/child relationship between a listbox and option in something like this example:

```html
<div role="listbox">
  <div tabindex-"-1">
    <button role="option">op</button>
  </div>
</div>
```

A `<div>` should never have a role of `presentation` unless it's explicitly applied. I think I remember some discussion around giving it a `role="group"` instead of `generic` based on some combination of naming & focused state, but I don't believe that's currently the case. If that were to change and `<div tabindex="-1">` were to be mapped as `group`, then my understanding is the above code would be disallowed.

Reply:
It's a good callout, and I think there are two potential ways forward:
1. The mapped role of a focusable `<div>` is no longer generic
2. Webkit allows this code to functionally work (note: this does not require them to remove the `<div>` from the a11y tree, just to not have it to disrupt the listbox/option relationship)

This came up some time ago in the ARIA WG discussions, and I think I remember @cookiecrook wanting to verify that this does not require UAs to remove generics from the a11y tree. I'll ask him to confirm though :)

Reply:
Some comments... 

1. I don't think "owned child" is a good term, especially when it can contain non-child descendants.
2. The proposed def does not count for shadow roots, which may also need accounting for? If so, in a separate issue.
3. Re: "The mapped role of a focusable `<div>` is no longer generic", @scottaohara may wish to address the mappings of `<div tabindex>` and `<div aria-globalattrs>` in separate rows for HTML-AAM?
4. Re: "Webkit allows this code to functionally work" IIRC this was a VoiceOver change. WebKit exposes the tree as it did, but VO (years ago) wouldn't step deeply into it if there were many layers of irrelevant container elements in between the listbox and its options.
5. I think all the code samples above are purposefully author ambiguous, so AT tries to do its best. If there's a specific "bug" or something the spec should address, let's call that out in an engine problem tracker or an ARIA issue. E.g. I think the focusable "text" role description is reasonable, as is labeling by the first descendant element ("op"), but we could discuss diffs of opinion in a new issue?

I'm hopeful I have responded to all points. If not, please call out again.